components/switch/test/switch-visibility.test.js

```diff
@@ -7,4 +8,9 @@ describe('d2l-switch-visibility', () => {
 		runConstructor('d2l-switch-visibility');
 	});
 
+	it('should have conditions if conditions are passed in', async() => {
+		const switchFixture = await fixture(html`<d2l-switch-visibility on>Conditions</d2l-switch-visibility>`);
+		expect(switchFixture._hasConditions).to.be.true;
```

Comment:
Not sure if this test is redundant given the new ["on-with-conditions"](https://github.com/BrightspaceUI/core/pull/2665/files#diff-a232a1a1541f68920c062087554cefc14fbd4897c80da754c035fc4337e416b6R32-R42) vdiff test that we will have.

Reply:
Would you want me to remove it then?

Reply:
Yup I think that's safe.